src/test/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalkerIntegrationTest.java

```diff
@@ -25,4 +25,18 @@ public void testExampleVariantWalker() throws IOException {
         );
         testSpec.executeTest("testExampleIntervalWalker", this);
     }
+
+    @Test
+    public void testExampleVariantWalkerWithSymbolicNameForReadInput() throws IOException {
```

Comment:
Should really be replaced with a `TaggedArgumentIntegrationTest` and a custom asserting walker, as suggested above.